deep_speech_2/test_ctc_beam_search_decoder.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import absolute_import
```

Comment:
Should we put it in a ./test folder？What is the best practice for a python unit test file?